e2/electron.vite.config.ts

```diff
@@ -3,6 +3,7 @@ import { nodePolyfills } from "vite-plugin-node-polyfills";
 import { svelte } from '@sveltejs/vite-plugin-svelte';
 import { sentryVitePlugin } from "@sentry/vite-plugin";
 import replace from '@rollup/plugin-replace';
+import { production } from '../app/logic/build';
```

Comment:
Is it ok to do like this? Because it will generate and upload the sourcemap whenever `yarn/npm build` executes.

Reply:
Yes, we make an exception for this `build.ts` file. We also use it in `backend.ts`.